consensus/src/detail/tests/processing_tests.rs

```diff
@@ -64,30 +64,66 @@ fn test_process_genesis_block() {
                 .blockchain_storage
                 .get_best_block_id()
                 .expect(ERR_BEST_BLOCK_NOT_FOUND),
-            Some(consensus.chain_config.genesis_block().get_id())
+            Some(consensus.chain_config.genesis_block_id())
         );
         assert_eq!(block_index.get_prev_block_id(), &None);
         assert_eq!(block_index.get_chain_trust(), 1);
         assert_eq!(block_index.get_block_height(), BlockHeight::new(0));
     });
 }
 
+// TODO: test the orphans' custom error hook
```

Comment:
Out of curiosity, why is the hook functionality implemented in this PR if there are no tests for it and it seems that the node doesn't even use it (`None` is given in `node/src/runner.rs`)

Reply:
Just for completeness of implementation as adding it wasn't much work, but I can imagine telling someone to do it can be misunderstood. It's much easier to ask someone to write tests for something simple than recreate something simple from scratch.